backend/src/main/java/com/google/blackswan/mock/SimpleAnomalyGenerator.java

```diff
@@ -104,13 +104,14 @@ private SimpleAnomalyGenerator(InputStream source, int threshold,
     }
 
     List<RelatedData> relatedDataList = SimpleRelatedDataGenerator.createGenerator()
-        .getRelatedData(metricName, dimensionName, 
+        .getRelatedData(DataInfo.of(metricName, dimensionName), 
                         listKeys.get(firstDataPointIndex),
                         listKeys.get(lastDataPointIndex));
 
     return new Anomaly(time, metricName, dimensionName, dataPoints, relatedDataList);
   }
 
+  /** TODO: Specify which metric and dimension data to look at for this generator. */
```

Comment:
is this the same TODO you had in the other PR (file systems PR)?

Reply:
yep